getting-started/interact-with-contract.md

```diff
@@ -0,0 +1,195 @@
+---
+order: 5
+---
+
+# Uploading and Interacting
+
+We have the binary ready. Now it is time to see some wasm action.
+You can use [Go CLI](#GO-CLI) or [Node Console](#Node-Console) as you wish.
+
+## GO CLI
+
+Before we upload the code, we need to compile the contract to binary.
+
+```bash
+# for the rest of this section, we assume you are in the same path as the rust contract (Cargo.toml)
+# compile wasm
+docker run --rm -v "$(pwd)":/code \
```

Comment:
Shouldn't this go in the previous section where we compile it (outside of docker)? Then all the rust/compile stuff and this is just working with the blockchain